packages/tooltip/src/LionTooltip.js

```diff
@@ -66,6 +66,14 @@ export class LionTooltip extends ArrowMixin(OverlayMixin(LitElement)) {
     });
   }
 
+  _hasDisabledInvoker() {
+    return (
+      this._overlayCtrl &&
+      this._overlayCtrl.invoker &&
+      /** @type {HTMLElement & { disabled: boolean }} */ (this._overlayCtrl.invoker).disabled
```

Comment:
could we also check for aria-disabled === 'true'?

Reply:
As in `&&` or `||`?

Reply:
Good point, I would assume ||